### ISSUE TITLE
Save template or recreate Retrospective

### ISSUE THREAD

**Issue by JaderMR** [microsoft/vsts-extension-retrospectives#389]
I want to save current model or recreate based on current one

**Comment by dsczltch** [comment]
We are also looking to create our own templates or to keep the same columns as the previous retro.

**Comment by tballew** [comment]
Yes, we normally add "What went well" and "What didn't go well" to the Start, Stop and Continue template, but we have to add it each sprint; it would be nice if you could save your changes as a template or at least preserve your changes to a template from sprint to sprint.

**Comment by JJKW1984** [comment]
Our organization would also love to see this feature implemented.

**Comment by dphanson63** [comment]
Copy retro is supported.
